### ISSUE TITLE
Add support for i18n of the landing page content

### ISSUE THREAD

**Issue by juho-jaakkola** [manacim/Landr#5]
Currently the page contents are the same regardless of which language the UI is using.

This is something that would ideally be fixed in Elgg core, but temporary workarounds are welcome.

**Comment by demyxco** [comment]
True. Feel free to add the setting content in the language.php file.

**Comment by juho-jaakkola** [comment]
Well, then there is no point in having a setting in the first place. :) (Which I personally do not mind, because I prefer doing stuff through code instead of settings in the first place.)

**Comment by demyxco** [comment]
What about the people who don't know how to code? Lol.

**Comment by demyxco** [comment]
Hey I was just wondering, doesn't Elgg handle i18n? Or did I not create a en.php file?

**Comment by juho-jaakkola** [comment]
I'm not sure what you mean.

Elgg does handle i18n when the `elgg_echo()` function is used, but not when the string comes from a plugin setting. This is why I think it would be better if the landing page contents would come from a translations file instead of plugin settings.

**Comment by demyxco** [comment]
Oh yeah, I need to update that.
